packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -281,8 +281,10 @@ public void onOpened(@NonNull CameraDevice device) {
           public void onClosed(@NonNull CameraDevice camera) {
             Log.i(TAG, "open | onClosed");
 
+            // Prevents calls to methods that would otherwise result in IllegalStateException exceptions.
+            cameraDevice = null;
+            closeCaptureSession();
             dartMessenger.sendCameraClosingEvent();
-            super.onClosed(camera);
```

Comment:
Why don't we need this?

Reply:
The default implementation of `onClosed` does nothing.  For callbacks, I'd be very surprised if there was an API that has a super implementation that does something. It'd be incorrect to call it a callback.
 https://developer.android.com/reference/android/hardware/camera2/CameraDevice.StateCallback#onClosed(android.hardware.camera2.CameraDevice)